spec/forms/candidate_interface/find_feedback_form_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+RSpec.describe CandidateInterface::FindFeedbackForm, type: :model do
+  let(:form) do
+    described_class.new(
+      path: '/course/T92/X130',
+      original_controller: 'courses',
```

Comment:
What is `original_controller`? Is this to do with with where the feedback request was initiated (i.e. Find)

Reply:
Yea maybe it should be called find_controller